string_regex_task_list_of_names.py

```diff
@@ -0,0 +1,9 @@
+# you have a list of variable names in camel case format
+# ["FirstItem", "FriendsList", "MyTuple"]
+# convert it to a list of variable names for python in snake case format
+# ["first_item", "friends_list", "my_tuple"]
+
+list_of_names = ["FirstItem", "FriendsList", "MyTuple"]
+string_of_names = ", ".join(list_of_names).lower()
+string_to_list = string_of_names.split(", ")
+print(string_to_list)
```

Comment:
![image](https://user-images.githubusercontent.com/25909810/215122688-f050f7b6-7d55-47ba-aa10-3e100607541e.png)
this is what I get after executing your code